File: Java/p11547.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DarKnight060198
 */
public class p11547 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int kasus = input.nextInt();
        for (int i = 0; i < kasus; i++) {
            int num = input.nextInt();
            String res = Integer.toString(automaticAnswer(num));
            System.out.println(res.charAt(res.length()-2));
        }
    }
    
    static int automaticAnswer (int num) {
        return ((((((num*567)/9)+ 7492)*235)/47)-498);
    }
}

File: C,C++/p11547.c
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

#include <stdio.h>

int pembalik (int);

int main () {
    int kasus, i;
    scanf("%d", &kasus);
    for (i = 0; i < kasus; i++) {
        int num;
        scanf("%d", &num);
        int res = num;
        res *= 567;
        res /= 9;
        res += 7492;
        res *= 235;
        res /= 47;
        res -= 498;
        if (res < 0) res *= -1;
        res %= 100;
        res /= 10;
        printf("%d\n", res);
    }
    
    return 0;
}
